apps/jobs/views.py

```diff
@@ -47,16 +47,16 @@
 
```

Comment:
@vkartik97 Your text editor has done a lot of unnecessary changes. can you please revert all those.